pkg/provider/aws/config.go

```diff
@@ -52,6 +52,7 @@ type GatewayType string
 // Enums for the types we allow in the aws provider so that we can reuse the same string within the provider
 const (
 	eks                                = "eks"
+	ecs                                = "ecs"
 	fargate                            = "fargate"
```

Comment:
Do we still need `fargate`?

Reply:
yeah if we dont then we would error out right now for anyone who has it in the config, i was going to leave it and cut another ticket to fully remove it later on so that we give them a little to switch the name for themselves. If we dont think thats important i should remove all notions of the naming.

Also im changing the code so that we only warn once about the naming change so need to do one more update

Reply:
I thought it gets re-written in https://github.com/klothoplatform/klotho/pull/160/files#diff-a3d824da3c42420cd5cbb0a4a2c0e7b5bfddd819652788a0596d195dc6e31fa5R153-R156 so it should never be `"fargate"`. But, if it's still needed, then no worries - keep it.

Reply:
ah right, good catch, let me remove